src/test/features/claim/helpers/featuresBuilder.ts

```diff
@@ -76,43 +135,39 @@ describe('FeaturesBuilder', () => {
   describe('Admissions Feature', () => {
     it('should add admissions to features if flag is set', async () => {
       mockFeatureFlag('cmc_admissions', true)
-      mockFeatureFlag('cmc_directions_questionnaire', false)
-      const features = await FeaturesBuilder.features(elegibleDraft, user)
+      const features = await FeaturesBuilder.features(eligibleDraft, user)
       expect(features).to.equal('admissions')
     })
   })
 
   describe('Directions Questionnaire Feature', () => {
-    it('should add dq to features if flag is set and draft is <= 300', async () => {
+    it('should add dq to features if flag is set and draft is <= 1000', async () => {
```

Comment:
Do you think this value is better as a constant so that if the value is changed it will be a smaller change?